pkgs/by-name/jo/journalist/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "journalist";
+  version = "1.0.0-unstable-2024-05-28";
+
+  src = fetchFromGitHub {
+    owner = "mrusme";
+    repo = "journalist";
+    rev = "eda6ce67c975f928ba2dfe94a39b69d3f864d051";
+    hash = "sha256-pxWBvh7j+IRvGLpQA0TEqorw7f0OZtXux5svMMqVooA=";
+  };
+
+  overrideModAttrs = _oldAttrs: {
+    postPatch = ''
+      cp ${./go.sum} go.sum
+    '';
+  };
+
+  vendorHash = "sha256-fEHVc9kRbeeXICWhJshLp9JK/ICBR/RB5SVChJzSXpI=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/mrusme/journalist/journalistd.VERSION=${version}"
+  ];
+
+  meta = with lib; {
+    description = "An RSS aggregator";
+    homepage = "https://github.com/mrusme/journalist";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ moraxyc ];
+    mainProgram = "journalist";
```

Comment:
```suggestion
    mainProgram = "journalist";
    platforms = with lib.platforms; unix ++ windows;
    broken = stdenv.isDarwin; # needing some apple_sdk packages
```

Reply:
Thanks for testing it on Darwin!

Reply:
i did not test it, but mark it as a broken until someone tests it. seams pretty reasonable default to me.

Reply:
and yeah, you have to import `stdenv` for this to work, i forget to include it in the suggestion, sorry

Reply:
I added it on top.

Language server is pretty good in this case😄